tests/test_cli_gui/test_search.py

```diff
@@ -139,7 +139,8 @@ def test_real_search():
     )
     assert response_list.status_code == 200
     data = json.loads(response_list.get_data(as_text=True))
-    assert len(data) == 9
+
```

Comment:
Updated this test which was failing after the introduction of webhook connection.